src/main/java/pairmatching/repository/MissionRepository.java

```diff
@@ -0,0 +1,30 @@
+package pairmatching.repository;
+
+import pairmatching.model.Level;
+import pairmatching.model.Mission;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MissionRepository {
+    private final List<Mission> missions = new ArrayList<>();
+
+    public void saveAllNamesWithLevel(List<String> missionNames, Level level) {
+        missions.addAll(missionNames.stream()
+                .map(missionName -> new Mission(level, missionName))
+                .collect(Collectors.toList()));
+    }
+
+    public List<String> findAllNamesByLevel(Level level) {
+        return missions.stream()
+                .filter(mission -> mission.getLevel() == level)
```

Comment:
```suggestion
                .filter(mission -> mission.checkLevel(level))
```
get 를 이용해 가져와서 비교한다기 보다 객체로 메세지를 보내 처리하는 방법도 좋을 것 같아요!!

Reply:
음! 감사합니다. :)